pkg/osquery/tables/pwpolicy/pwpolicy.go

```diff
@@ -0,0 +1,133 @@
+//+build darwin
+
+package pwpolicy
+
+import (
+	"bytes"
+	"context"
+	"os/exec"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+type Table struct {
+	client       *osquery.ExtensionManagerClient
+	logger       log.Logger
+	tableName    string
```

Comment:
Don't really need this for a one off like this. You can hardcode it on line 46

Reply:
ended up keeping it since I saw this pattern again in ioreg.